tiffany-tiffanyteaze/week-3/giffaw/scripts/app.js

```diff
@@ -0,0 +1,31 @@
+let giphyUrl = "http://api.giphy.com/v1/gifs/search?q=";
+let apiKey = 'mIyOSO9AfIEquW3Daf5YzSdJ5PmOvG6R';
+
+$( document ).ready(function() {
+  $('form').on('submit', function (e) {
+    e.preventDefault();
+    console.log("You clicked submit.");
+    console.log($('.gif-input').val());
+
+    let input = $('.gif-input').val();
+
+  $.ajax ({
+    method: 'GET',
+    url: giphyUrl + input + "&api_key=" + apiKey,
```

Comment:
Very nice breakdown of the url. It is semantic and easy to follow. My usual set it would be to save all of that into a variable and reference that variable in the ajax call, but that is just my way of organizing data. 
```js
let finalUrl = `${giphyUrl}${input}&api_key=${apiKey}`

$.ajax({
url: finalUrl
})
```